kubera/install/experiments/assert-kubera-svc.yaml

```diff
@@ -0,0 +1,219 @@
+apiVersion: dope.mayadata.io/v1
+kind: Recipe
+metadata:
+  name: assert-kubera-svc-running
```

Comment:
Make the names simpler
Simple does not mean abbreviated
For example `assert-if-kubera-services-are-running`